### PR TITLE
release object scale 0.63

### PR DESCRIPTION
**Pull request by garretwu**
## Purpose
[OBSDEF-4756](https://jira.cec.lab.emc.com:8443/browse/OBSDEF-4756)
_List the changes for this PR_

## PR checklist
- [x] make test passed
- [x] make build passed
- [x] helm install <chart> passed
- [x] vSphere7 make package deployments passed
- [x] helm upgrade <chart> passed
- [ ] helm test <release_name> passed

## Testing
_Paste the output of your helm install/vSphere7 deployment testing here_

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -39,8 +39,8 @@ HELM_MONITORING_ARGS = # --set global.monitoring.tag=${YOUR_VERSION_HERE}
 HELM_UI_ARGS         = # --set image.tag=${YOUR_VERSION_HERE}
 HELM_GRAPHQL_ARGS    = # --set objectscale-graphql.tag=${YOUR_VERSION_HERE}
 HELM_INSTALLER_ARGS  = # --set objectscale-graphql.helm-controller.tag=${YOUR_VERSION_HERE}
-HELM_DECKS_ARGS      = # --set image.tag=${YOUR_VERSION_HERE}
-HELM_KAHM_ARGS       = # --set image.tag=${YOUR_VERSION_HERE}
+HELM_DECKS_ARGS      =  --set image.tag=${0.62}
+HELM_KAHM_ARGS       =  --set image.tag=${0.62}
```
Comment (eddiepavkovic) [inline Makefile:43]: No don't make the change here.   What I meant was change the make release to bypass 
Decks and kam versions for this. Build.

Comment (garretwu) [reply Makefile:43]: not quiet understand.
release target is as follows:
release: decksver flexver build-all generate-issues-events-all add-to-git

do you mean I need to update decksver target and skip "decks kahm" in DECKSCHARTS?

Comment (eddiepavkovic) [reply Makefile:43]: undo any changes in the Makefile.  i'll just retag all the decks/kahm/service-tools 62 images to 63 so that there's no confusion.

Comment (garretwu) [reply Makefile]: Done

**Comment by eddiepavkovic** [comment]
ok good.  please do the following for the build.

1. Run `make release` to build out the charts
2. Add people to the reviewers list so that you can get 2 approvals.  
3. Fill out the checklist above
4. add yourself as assignee
5. Fill in the testing section

good luck.

File: Makefile [hunk 2]
```diff
@@ -8,7 +8,7 @@ DECKSCHARTS := decks kahm supportassist service-pod dellemc-license decks-suppor
 FLEXCHARTS := ecs-cluster objectscale-manager objectscale-vsphere objectscale-graphql helm-controller iam statefuldaemonset-operator influxdb-operator federation logging-injector
```
Comment (vazhem) [inline Makefile:9]: Please merge latest commits from master to your branch.
Unsure that 32115e6c95e37c1588d1cb0e8c585c0c021569a1 commit is present in your branch before you perform make release to build and tag charts

**Review by vazhem — CHANGES_REQUESTED** [review]

**Review by fedor-chemashkin — APPROVED** [review]

**Review by vazhem — APPROVED** [review]

**Pull request merged**